core/services/chainlink/application.go

```diff
@@ -677,6 +677,19 @@ func (app *ChainlinkApplication) RunJobV2(
 			CreatedAt:      t,
 			FinishedAt:     &t,
 		}, nil, false)
+		elapsed := time.Since(t)
```

Comment:
Feels icky having all this logic in `application.go` - can it move into the runner (I mean, the meat of this entire function)?